frontend/src/scenes/session-recordings/apm/performanceEventDataLogic.ts

```diff
@@ -24,14 +24,19 @@ export interface PerformanceEventDataLogicProps extends SessionRecordingDataLogi
     key?: string
 }
 
+/** it's pretty quick to sort an already sorted list */
+function sortPerformanceEvents(events: PerformanceEvent[]): PerformanceEvent[] {
+    return events.sort((a, b) => (a.timestamp.valueOf() > b.timestamp.valueOf() ? 1 : -1))
+}
+
 /**
  * If we have paint events we should add them to the appropriate navigation event
  * this makes it easier to draw performance cards for navigation events
  */
 function matchPaintEvents(performanceEvents: PerformanceEvent[]): PerformanceEvent[] {
-    // KLUDGE: this assumes that the input is sorted by timestamp and relies on the identity of the events to mutate them
+    // NB: this relies on the input being sorted by timestamp and relies on the identity of the events to mutate them
```

Comment:
What NB means?

Reply:
Ah... it's latin "Nota bene"

https://en.wikipedia.org/wiki/Nota_bene

Reply:
TIL